content/pt-br/docs/reference/setup-tools/kubeadm/generated/kubeadm_upgrade_node.md

```diff
@@ -0,0 +1,115 @@
+<!--
+The file is auto-generated from the Go source code of the component using a generic
+[generator](https://github.com/kubernetes-sigs/reference-docs/). To learn how
+to generate the reference documentation, please read
+[Contributing to the reference documentation](/docs/contribute/generate-ref-docs/).
+To update the reference content, please follow the 
+[Contributing upstream](/docs/contribute/generate-ref-docs/contribute-upstream/)
+guide. You can file document formatting bugs against the
+[reference-docs](https://github.com/kubernetes-sigs/reference-docs/) project.
+-->
+
+Comando para atualização de um nó no cluster
+
+### Sinopse
+
+Comando para atualização de um nó no cluster
+
+O comando "node" executa as seguintes fases:
+
+```
+preflight       Executa as verificações de atualização pre-flight do nó
```

Comment:
Perhaps "verificações pré-atualização" (as pre-flight here seems to mean "before doing an action")?
Let me know what you think.